content/en/metrics/overview.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Metrics Overview Page
+aliases:
+  - /metrics/faq/metrics-without-limits/
+  - /metrics/guide/metrics-without-limits-getting-started/
+further_reading:
+  - link: "https://www.datadoghq.com/blog/metrics-without-limits"
+    tag: "Blog"
+    text: "Dynamically control custom metrics volume with Metrics without Limits™"
+  - link: "https://dtdg.co/fe"
+    tag: "Foundation Enablement"
+    text: "Join an interactive session to unlock the full potential of metrics"
+---
+
+## Overview
+
+The Metrics Overview page provides you with a deeper understanding of your metrics landscape, and is helpful for users of all experience levels. Explore the sources of your metrics, generate additional metrics from Datadog products, and enable Metrics without Limits™ and historical data ingestion.
+
+The sections below detail the different parts of the page and provide guidance on how you can maximize the value of your Datadog metrics.
+
+## How your metrics flow through Datadog
+
+{{< img src="metrics/overview/how_metrics_flow.png" alt="The how your metrics flow through Datadog section of the metrics overview page" >}}
+
+The topmost section of the page shows all of your metric sources, along with any configurable processing and a breakdown of the metrics by type (standard or [custom][1]).
+
+### Metric Sources
+
+The **Metric Sources** column shows a summary of the metric sources reporting to Datadog. Click on any of the sources to open the [Summary page][2] scoped to that source. Your Datadog metrics can originate from the following sources:
+
+{{% collapse-content title="Datadog Agent" level="h4" %}}
+The [Datadog Agent][3] collects metrics from the hosts it's installed on, and forwards them to Datadog. These metrics could originate from:
+
+   - Any of the official Datadog integrations that are bundled with the Agent. See the [integrations-core repository][4] for a full listing of the available integrations, and [Integration Management][5] for more information about managing these integrations.
+   - [DogStatsD][6], a metrics aggregation service bundled with the Datadog Agent. DogStatsD implements the [StatsD][7] protocol with some Datadog-specific extensions.
+   - [Custom checks][8], which are used to collect metrics from custom applications or unique systems. You define the logic for the check yourself in the Agent's configuration files. See [Writing a Custom Agent Check][9] for more information.
+   - [Marketplace integrations][10] installed on the Agent. The [Datadog Marketplace][11] is a digital marketplace where Technology Partners can list their paid offerings to Datadog users.
+{{% /collapse-content %}}
+
+{{% collapse-content title="Cloud integrations" level="h4" %}}
```

Comment:
@buraizu should we capitalize "Integrations"?

Reply:
+1

Reply:
Unless part of an H1 header, titles are written in sentence case ([style guide](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#headers) for reference)